pinot-broker/src/main/java/com/linkedin/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -269,6 +288,8 @@ public BrokerResponse handleRequest(JSONObject request, @Nullable RequesterIdent
     // Set total query processing time
     long totalTimeMs = TimeUnit.NANOSECONDS.toMillis(executionEndTimeNs - compilationStartTimeNs);
     brokerResponse.setTimeUsedMs(totalTimeMs);
+    requestStatistics.setQueryProcessingTime(totalTimeMs);
+    requestStatistics.setStatistics(brokerResponse);
```

Comment:
This is already  being set in line 131

Reply:
The one in line 131 is set using brokerRequest (and this one is from brokerResponse).